lib/tasks/linksf.rake

```diff
@@ -50,16 +50,16 @@ namespace :linksf do
 
       puts 'resource description is :' + resource.long_description
 
+      if resource.long_description.blank? || resource.long_description.length < 15
+        puts 'replacing bad description with nil'
+        resource.long_description = nil
+      end
+
       record[:services].each do |service_json|
         category_name = service_json[:category]
         category_name = 'shelter' if category_name == 'housing'
         cat = Category.where('lower(name) = ?', category_name).first
 
-        if resource.long_description.blank? || resource.long_description.length < 15
-          puts 'replacing bad description with nil'
-          resource.long_description = nil
-        end
-
         resource.categories << cat
```

Comment:
Good question, a resource could have multiple categories now, answer to this is unclear to me. 

This would depend on how we want to use this information I think?

@jjfreund